src/blue.ts

```diff
@@ -234,13 +235,16 @@ export function blueProxyFactory(env: MembraneBroker) {
             if (isUndefined(redDescriptor)) {
                 // looking in the blue proto chain to avoid switching sides
                 const blueProto = getBlueValue(ReflectGetPrototypeOf(target));
+                if (isNull(blueProto)) {
+                    return;
+                }
                 return ReflectGet(blueProto, key, receiver);
             }
             if (hasOwnProperty(redDescriptor, 'get')) {
                 // Knowing that it is an own getter, we can't still not use Reflect.get
                 // because there might be a distortion for such getter, and from the blue
                 // side, we should not be subject to those distortions.
-                return apply(getBlueValue(redDescriptor.get), receiver, emptyArray);
+                return apply(getBlueValue(redDescriptor.get!), receiver, emptyArray);
```

Comment:
since the descriptor is a result of reflect.getownpropertydescriptor, if the get is an own property of the descriptor, the get is a function, not need to check it, hence the usage of `!` here to avoid an extra if condition.